src/test/java/TaskServiceTest.java

```diff
@@ -0,0 +1,78 @@
+import algorithmic_task.Task;
+import algorithmic_task.TaskService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TaskServiceTest {
+
+    private static TaskService taskService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        taskService = new TaskService(new Task());
+    }
+
+    @Test
+    public void areSimilar_differentSentences_true() {
+
+        String s1 = "hwefhewf in house";
+        String s2 = "mom in home";
+        boolean areSimilar = taskService.areSimilar(s1, s2).getResult();
+
+        assertTrue(areSimilar);
+    }
+
+    @Test
+    public void areSimilar_oneWordMultiTime_true() {
+
+        String s1 = "house home home";
+        String s2 = "house house house";
+        boolean areSimilar = taskService.areSimilar(s1, s2).getResult();
+
+        assertTrue(areSimilar);
+    }
+
+    @Test
+    public void areSimilar_InvertedOrderOfTheSentence_false() {
+
+        String s1 = "mother in house";
+        String s2 = "in house mom";
+        boolean areSimilar = taskService.areSimilar(s1, s2).getResult();
+
+        assertFalse(areSimilar);
+    }
+
+    @Test
+    public void areSimilar_differentLengthOfSentences_false() {
+
+        String s1 = "mother in house d sf sdfs f";
+        String s2 = "in house mom";
+        boolean areSimilar = taskService.areSimilar(s1, s2).getResult();
+
+        assertFalse(areSimilar);
+    }
+
+    @Test
+    public void areSimilar_emptySentences_False() {
+
```

Comment:
I am not sure that empty sentences are not similar. Could you explain this?

Reply:
In my opinion: In instruction of this task is information that sentences are considered as similar, if corresponding words from sentences are in the same set. Empty sentences have no words so they are not similar.